### PR TITLE
Fix/optimise/add typography/format/toolbar icons/improve metadata

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1204.

### CODE REVIEW

File: icons/heading-circle.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "danielbayley"
+  ],
+  "tags": [
+    "hospital",
+    "accident",
+    "emergency",
+    "helicopter pad",
+    "helipad"
```
Comment (jguddas) [inline icons/heading-circle.json:11]: Maybe, idk.
```suggestion
    "helipad",
    "help"
```

**Comment by jguddas** [comment]
Why are there so many changes in this one PR?

**Comment by danielbayley** [comment]
> Not quite sure about the use cases for the typography+`-square/-circle` icons. 🤔

For example a toolbar to toggle the respective features…

**Comment by karsa-mistmere** [comment]
> @karsa-mistmere For example use in a toolbar to toggle the respective features…

Yes, but why the need for a square or circle? Wouldn't the base icon make more sense in a toolbar?

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**